src/Components/Server/src/BlazorPack/BlazorPackHubProtocol.cs

```diff
@@ -457,17 +445,6 @@ private static object DeserializeObject(ref MessagePackReader reader, Type type,
                 {
                     return reader.ReadInt64();
                 }
-                else if (type == typeof(float))
```

Comment:
Sort of unrelated, why don't you use `is` for these checks?

Reply:
It's a `Type` instance itself not an instance of an object.

Reply:
I thought `is` only works on instances, such as `someInstance is float`, not when comparing two `Type` instances. Did I get that wrong?

Reply:
Ah, gotcha